src/cljs/chronograph_web/pages/timers/views.cljs

```diff
@@ -48,19 +48,30 @@
                                   [::timers-events/stop-timer id]
                                   [::timers-events/delete-timer id]]]))}]))
 
+(defn actions [showing-create-timer-widget?]
+  [:<>
+   [antd/button
+    {:icon    icons/ArrowLeftOutlined
+     :onClick #(rf/dispatch [::timers-events/modify-selected-date -1])}]
+   [antd/button
```

Comment:
I would put the calendar in between the buttons, but I won't insist on it.

Reply:
A minor touch of polish could be to add tooltips over the buttons that say "Next day" and "Previous day", since they are just icons currently.